### PR TITLE
PMNT-1791: Add Bancontact payment method

### PR DESCRIPTION
**Pull request by shawncruz**
Depends on https://github.com/flowcommerce/aws-s3-public/pull/146

### CODE REVIEW

File: data/final/payment-methods.json [hunk 1]
```diff
@@ -131,6 +131,32 @@
...
   },
+  {
+    "id": "bancontact-mobile",
+    "type": "online",
+    "name": "Bancontact app",
+    "images": {
+      "small": {
+        "url": "https://flowcdn.io/util/logos/payment-methods/bancontact/30/original.png",
+        "width": 65,
+        "height": 41
```
Comment (shawncruz) [inline data/final/payment-methods.json:142]: These heights are incorrect, however this seems to be standard across all payment method images. @RoryH Do you have any context here?

File: data/final/payment-methods.json [hunk 2]
```diff
@@ -131,6 +131,32 @@
     ],
     "capabilities": []
   },
+  {
+    "id": "bancontact-mobile",
```
Comment (shawncruz) [inline data/final/payment-methods.json:135]: This is technically a "flow-defined ID" in that it doesn't match [Adyen's](https://docs.adyen.com/development-resources/paymentmethodvariant), however we could come up with a more "standard" scheme. Two ideas I had:
1. Have a template like `flow_payment_method_[snake cased type]`, e.g. `flow_payment_method_bancontact_mobile`
2. Have a generated scheme like we do in other parts of the code like fpm-[uuid]

Really what seems to matter the most is that we don't have a one to one mapping between this ID and the Adyen defined payment method type.

@amcguiga Thoughts?

Comment (amcguiga) [reply data/final/payment-methods.json:135]: I'm debating whether this should just be considered as `bancontact`. From a consumer branding perspective, the two integration approaches are functionally identical - it's the same logo and the same payment guts under the hood, regardless of whether it's used in a `card_authorization` or an `online_authorization` context.

Comment (shawncruz) [reply data/final/payment-methods.json:135]: Does this introduce an issue if we ever try to map our flow ID to Adyen's `bcmc` or `bcmc_mobile` brand codes?

Comment (shawncruz) [reply data/final/payment-methods.json:135]: For now, I'll keep it as `bancontact`

Comment (amcguiga) [reply data/final/payment-methods.json]: No, classifying them as the same payment method in the auth is fine, as that's the business criteria of the transaction from a pricing, settlement, and chargeback perspective. 

If we wanted to support both technical flows, we could process them differently by starting the auth as a `card_authorization` vs `online_authorization` discriminator, as those would naturally flow into different paths.

**Comment by shawncruz** [comment]
Blocked by https://github.com/flowcommerce/json-reference/pull/116

**Review by amcguiga — APPROVED** [review]

**Pull request merged**